app/src/main/res/animator/pocket_video_item.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yayayayayay my favorite part - I'm glad this is so easy. Looking forward to seeing this in the home tiles too.